lib/py/src/ext/protocol.tcc

```diff
@@ -431,6 +433,7 @@ bool ProtocolBase<Impl>::encodeValue(PyObject* value, TType type, PyObject* type
     }
```

Comment:
I think we need DECREF before the "return false" above too.
Feel free to use ScopedPyObject (or not).